spec/index.md

```diff
@@ -1835,6 +1835,16 @@ Indicates various information about the canister. It contains:
 
 -   The cycle balance of the canister.
 
+-   Statistics regarding the query execution of the canister: a tuple `(num_queries, num_instructions, num_request_payload_bytes, num_response_payload_bytes)` containing
```

Comment:
@Dfinity-Bjoern - this is where we are planning to make query stats available to developers.